apps/pipeline/upstream/third-party/mysql/base/mysql-pv-claim.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: mysql-pv-claim
+  namespace: kubeflow 
```

Comment:
Namespace is added by kustomization.yaml file: https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/env/dev/kustomization.yaml#L19

Reply:
Took the latest manifest repo and build it and checked; It is coming proper. Sorry maybe I had the older repo